WcaOnRails/config/locales/en.yml

```diff
@@ -747,6 +748,9 @@ en:
       privacy: "Privacy"
       disclaimer: "Disclaimer"
       find_us: "Find us"
+  page:
```

Comment:
If we don't have any other pages that have localized titles, you don't have to start doing it now.

Reply:
We have more coming - like Board & Officers page also have localized title.